Misc/NEWS

```diff
@@ -364,6 +364,8 @@ Library
 - bpo-29534: Fixed different behaviour of Decimal.from_float()
   for _decimal and _pydecimal. Thanks Andrew Nester.
 
+- bpo-10379: Deprecate locale.format in lue of locale.format_string
```

Comment:
Google says it's spelled 'lieu'.  Also the entry should note that the monetary argument was added to format_string.  So, how about "locale.format_string now supports the 'monetary' keyword argument, and locale.format is deprecated."